fairlearn/metrics/_plot_model_comparison.py

```diff
@@ -119,6 +120,8 @@ def plot_model_comparison(
         zeros((len(y_true), 1)),  # Dummy values for X
         y=y_true,
         sensitive_features=sensitive_features,
+        # TODO: remove this once we supports all narwhals inputs in this module
+        default_backend=nw.Implementation.PANDAS,
```

Comment:
I had to add this in places were `narwhals` is not yet supported. Since we import `polars` in the tests, it becomes the default backend.